src/utils/Context.js

```diff
@@ -27,7 +38,15 @@ export const ContextProvider = ({ children }) => {
         },
       })
     },
+    reset: () => {
+      localStorage.removeItem(stateKey)
+    },
   })
 
+  useEffect(() => {
+    const { update, ...storedState } = state
+    localStorage.setItem(stateKey, JSON.stringify(storedState))
```

Comment:
really loving local storage, simple - works, not the cookie cash madness and handling...

Reply:
main issue is it blocks the main thread and isn't accessible from web workers. Other than that it's great